tests/pass/shims/x86/intrinsics-x86-adx.rs

```diff
@@ -1,12 +1,7 @@
 // Ignore everything except x86 and x86_64
 // Any new targets that are added to CI should be ignored here.
 // (We cannot use `cfg`-based tricks here since the `target-feature` flags below only work on x86.)
-//@ignore-target-aarch64
-//@ignore-target-arm
-//@ignore-target-avr
-//@ignore-target-s390x
-//@ignore-target-thumbv7em
-//@ignore-target-wasm
+//@ignore-target: aarch64 arm avr s390x thumbv7em wasm
```

Comment:
neat side effect of disjunctive `only` filters: disjunctive `ignore` filters

Reply:
ah we should probably switch to `only` here: https://github.com/rust-lang/miri/pull/3492#discussion_r1575960234

Reply:
For `ignore`, this is equivalent to the old form, right?

I wonder if it would be worth to make `ignore-target: x86 linux` mean "ignore targets that contain both of these substrings"... but it's probably too confusing and that seems rarely needed.

Reply:
And yes this should probably be `only-target: x86 i686` now.